midnight/proto/midnight_state.proto

```
@@ -0,0 +1,238 @@
+// NOTE:
+// This proto definition is intended to be moved into a separate package/crate
+// in the future. The goal is to publish it as a reusable dependency so that
+// external clients can import and depend on the shared API contract directly.
+syntax = "proto3";
+
+package midnight_state;
```

Buf (1.70.0)

[error] Files with package "midnight_state" must be within a directory "midnight_state" relative to root but were in directory "midnight/proto".

(PACKAGE_DIRECTORY_MATCH)